libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -221,23 +221,23 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 
     // @Param: _POS1_X
     // @DisplayName: Antenna X position offset
-    // @Description: X position of the first GPS antenna in body frame. Positive X is forward of the origin. Use antenna phase centroid location if provided by the manufacturer.
+    // @Description: X position of the first GPS antenna in body frame with the origin defined to be the center of control (center of gravity for aerial vehicles). Positive X is forward of the origin. Use antenna phase centroid location if provided by the manufacturer.
```

Comment:
The EKF's body frame origin is at the IMU unless the IMU body frame position parameters https://ardupilot.org/plane/docs/parameters.html#ins-pos1-x-imu-accelerometer-x-position have been set.